service/dap/server.go

```diff
@@ -1702,7 +1741,7 @@ func (s *Server) convertVariableWithOpts(v *proc.Variable, qualifiedNameOrExpr s
 		if v.Len > int64(len(v.Children)) { // Not fully loaded
 			if v.Base != 0 && len(v.Children) == 0 { // Fully missing
 				value = reloadVariable(v, qualifiedNameOrExpr)
-			} else { // Partially missing (TODO)
+			} else if includeLoadedInfo {
```

Comment:
FYI It would be more clear what's going on if you could reference the paging capability here directly. I see you have it as an option because of the expression evaluation. It's not clear to me why - please see question below.

Also, all the other reloading happens in this function. If you cannot do the same for arrays/slices, it would be helpful to add a comment here with some insights.

Reply:
I added the reference to the paging capability here and added an explanation comment about